apps/functions-subscription/src/adapters/azure/servicebus/__tests__/event.test.ts

```diff
@@ -28,16 +33,96 @@ describe('makeEventWriterServiceBus', () => {
   });
 
   it('should return an error in case of failure', async () => {
-    const mock = makeServiceBusMock();
-    const client = mock as unknown as ServiceBusSender;
+    const { sender: client } = makeServiceBusMocks();
     const error = new Error('Oh No!');
 
-    mock.sendMessages.mockRejectedValueOnce(error);
+    client.sendMessages.mockRejectedValueOnce(error);
 
     const actual =
       await makeEventWriterServiceBus(client).send(aSubscription)();
 
     expect(actual).toStrictEqual(E.left(error));
-    expect(mock.sendMessages).toBeCalledTimes(1);
+    expect(client.sendMessages).toBeCalledTimes(1);
```

Comment:
**note**: changes here are related to [this](https://github.com/pagopa/trial-system/pull/109/files#r1662564766) comment